src/base/logger.cpp

```diff
@@ -35,9 +35,9 @@ void Logger::freeInstance()
 
 void Logger::addMessage(const QString &message, const Log::MsgType &type)
 {
-    QWriteLocker locker(&lock);
+    QWriteLocker locker(&m_lock);
 
-    Log::Msg temp = { msgCounter++, QDateTime::currentMSecsSinceEpoch(), type, message.toHtmlEscaped() };
+    Log::Msg temp = { m_msgCounter++, QDateTime::currentMSecsSinceEpoch(), type, message.toHtmlEscaped() };
```

Comment:
There should not be spaces inside braces in case of initializers (I mean spaces after start brace and before end brace).